diff --git a/dist/pool/index.mjs b/dist/pool/index.mjs
index 3375101c28742ab7facbc0c0642fffa004ba883d..57d4a4a3ccc09322d733733e9bddb40d0ac108ba 100644
--- a/dist/pool/index.mjs
+++ b/dist/pool/index.mjs
@@ -11733,7 +11733,7 @@ async function handleModuleFallbackRequest(vite, request) {
 //#region src/pool/config.ts
 const PLUGIN_VALUES = Object.values(PLUGINS);
 const WorkersPoolOptionsSchema = z.object({
-	main: z.ostring(),
+	main: z.string().optional(),
 	remoteBindings: z.boolean().default(true),
 	additionalExports: z.record(z.string(), z.union([
 		z.literal("WorkerEntrypoint"),
@@ -11742,8 +11742,8 @@ const WorkersPoolOptionsSchema = z.object({
 	])).default({}),
 	miniflare: z.object({ workers: z.array(z.object({}).passthrough()).optional() }).passthrough().optional(),
 	wrangler: z.object({
-		configPath: z.ostring(),
-		environment: z.ostring()
+		configPath: z.string().optional(),
+		environment: z.string().optional()
 	}).optional()
 });
 function isZodErrorLike(value) {
